core/arch/arm/plat-d02/conf.mk

```diff
@@ -4,6 +4,7 @@ $(call force,CFG_TEE_CORE_NB_CORE,16)
 CFG_NUM_THREADS ?= 16
 CFG_CRYPTO_WITH_CE ?= y
 CFG_WITH_STACK_CANARIES ?= y
+CFG_CORE_STACK_PROTECTOR_STRONG ?= y
```

Comment:
I can move it, just curious, what is our standpoint here? > 50% of platform having a feature enabled means default to `mk/config.mk`? Or is it only when it is 100%? I checked `CFG_WITH_STACK_CANARIES` when I did this and that seem to have it in all `plat-xyz` files. Sounds like this is something that should be mentioned in our platform porting guidelines.

Reply:
I agree that we are lacking some consistency here ;-)

Generally speaking, if a feature is not platform-specific, I think it deserves to be documented in `mk/config.mk` and set to a sensible value at the same time. If we think that most platform should have it enabled by default, then it is `?= y`. Otherwise, `?= n`. The platform config always takes precedence anyway.

Reply:
Makes sense, I'll update the platform guidelines to also mention a few words about that when doing the optee_docs updates.

Reply:
Fix ready, will push soon.